### PR TITLE
added test coverage for -notcontains, -is, -isnot operators

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
-like and -contains already has coverage

Fix https://github.com/PowerShell/PowerShell/issues/4162

### CODE REVIEW

File: test/powershell/Language/Operators/ComparisonOperator.Tests.ps1 [hunk 1]
```diff
@@ -47,9 +47,13 @@ Describe "ComparisonOperator" -tag "CI" {
 	 !0       | Should Be $true
     }
 
-	It "Should be $true for 'Hello','world' -contains 'Hello'" {
-	 $arr= 'Hello','world'
-	 $arr -contains 'Hello'       | Should Be $true
+	It "Should be <result> for 'Hello','world' <operator> <rhs>" -TestCases @(
+        @{result = $true; operator = "-contains"; rhs = "'Hello'"},
```
Comment (adityapatwardhan) [inline test/powershell/Language/Operators/ComparisonOperator.Tests.ps1]: Test for case sensitive variants? ccontains and cnotcontains? There are some tests below. Can we move them to this It and add it as part of -TestCases array.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: test/powershell/Language/Operators/ComparisonOperator.Tests.ps1 (hunk 1 above)
Comment (SteveL-MSFT) [inline test/powershell/Language/Operators/ComparisonOperator.Tests.ps1]: I'll do some general cleanup of the tests

**Comment by SteveL-MSFT** [comment]
@adityapatwardhan addressed

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Pull request merged**